File: Stage3/Python/Problem10950.py
def main():
    T = int(input())
    results = []
    
    for _ in range(T):
        A, B = map(int, input().split())
        results.append(A + B)
    
    for i in range(T):
        print(results[i])
    
if __name__ == "__main__":
    main()

File: Stage3/C/Problem10950.c
#include <stdio.h>

int main() {
    int A, B, T;

    scanf("%d", &T);
    int results[T];

    for (int i = 0; i < T; i++) {
        scanf("%d %d", &A, &B);
        results[i] = A + B;
    }

    for (int i = 0; i < T; i++) {
        printf("%d\n", results[i]);
    }

    return 0;
}